### DISCUSSION TITLE
VHDK and VirtualBox

### DISCUSSION THREAD

**Discussion by nmz787** [SmoothWAN/SmoothWAN#44] [General & Help]
Hi, I want to try out the x86 VHDK image. I have it running, but I can't access the IP (172.17.17.2, from the VM's ifconfig output for `br-lan`) from my VM host. I'm not sure what network adapter settings to use (NAT, bridged, etc). My PC has two separate ethernet interfaces, as well as a wlan interface. My modems are both USB.

P.S. I have a raspi 400 kit coming in a week or so, and will transition to that ASAP.

**Comment by TalalMash** [comment]
Hi, unfortunately the latest VirtualBox is a bit tricky due to unordered interfaces start up.
Simply set the bridged LAN interface to "Paravirtualized virtual adapter", it will come up first and OpenWRT will use it for LAN. While the rest and WANs untouched/default.
All adapters should be set as bridged.
Note that the order will change for the multiple WANs if you add more interfaces which affects static IP configuration needing update.
My example:
Adapter 1 LAN, Adapter 2 Wi-Fi, Adapter 3 Android tether
![image](https://user-images.githubusercontent.com/96490382/178232320-3e3d259a-df13-4b75-ace0-f0846500c602.png)
![image](https://user-images.githubusercontent.com/96490382/178232371-4beb1a42-13f5-46f2-b56e-05697c39bf83.png)
![image](https://user-images.githubusercontent.com/96490382/178232418-506af610-7c20-420c-a186-6a3e75ebd29a.png)

Thanks for the report.
